lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -240,21 +240,61 @@ void SessionState::set_current_version(
   if (session_uc) {
     session_uc->is_current_version_updated = true;
     session_uc->updated_current_version    = new_session_version;
+    MLOG(MINFO) << " Current version is " << get_current_version();
```

Comment:
demote to MDEBUG?

Reply:
Yes, missed the debug level change. Now fixed it.